io-engine/src/subsys/config/opts.rs

```diff
@@ -97,6 +97,8 @@ pub struct NvmfTgtConfig {
     pub opts: NvmfTcpTransportOpts,
```

Comment:
Do we need NvmfRdmaTransportOpts?

Reply:
> Do we need NvmfRdmaTransportOpts?

Yes, that is part of the upcoming changes.